tests/kernels/test_flash_attn.py

```diff
@@ -126,7 +127,8 @@ def test_flash_attn_with_paged_kv(
         cache_seqlens=kv_lens_tensor,
         softcap=soft_cap if soft_cap is not None else 0,
         window_size=window_size,
-    ).squeeze(1)
+        out=output.unsqueeze(1),
```

Comment:
suggestion - do the unsqueeze during creation (either by passing in the modified shape or just `.empty_like(...).unsqueeze(1)`. I think that will be cleaner